tests/lib/rules/newline-before-return.js

```diff
@@ -111,78 +111,149 @@ ruleTester.run("newline-before-return", rule, {
 
     invalid: [
         {
+            code: "function a() {\nvar b; return;\n}",
+            errors: ["Expected newline before return statement."],
+            output: "function a() {\nvar b; \nreturn;\n}"
```

Comment:
Indeed, the idea was to rely on the multipass for this one. But let me take a look if I can make the fixer a bit smarter to do it in one pass!